util/memfault_sdk/pkg.yml

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+pkg.name: util/memfault_sdk
+pkg.description: "Memfault SDK package"
+pkg.author: "Apache Mynewt <dev@mynewt.apache.org>"
+pkg.homepage: "http://mynewt.apache.org/"
+pkg.keywords:
+
+pkg.deps:
+    - "@apache-mynewt-core/kernel/os"
+
+pkg.deps.MEMFAULT_ENABLE:
+    - "@mynewt-memfault-sdk/memfault-firmware-sdk/ports/mynewt"
```

Comment:
Shouldn this be: `@proxyco/mynewt-memfault-sdk/ports/mynewt` ?

Reply:
I set it up to be similar to tinyusb where a readme outlines the requirement to add the repo to `project.yml` 

`@proxyco/` also needs to be added to `project.yml` for any project using this

Reply:
That is correct. It is a name of the repo, so, can be anything. In mynewt since all components come from mynewt, it is has the mynewt prefix. If we wanted this to be ‘proxyco-memfault-sdk’, that is fine too.

Reply:
I added the readme to the package to match how tiny usb did it. I think it should suffice